pkg/util/converter/prom.go

```diff
@@ -417,7 +418,6 @@ l1Fields:
 						timeField.Append(ts)
 
 					case "labels":
-						max := 0
 						pairs, err := readLabelsAsPairs(iter)
```

Comment:
not really related to this PR, but one thing i just noticed while reviewing this locally is that we don't seem to handle an exemplar without labels.